app/test_main.py

```diff
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        pytest.param(0, 14, [0, 0], id="animals' age is less then 15"),
```

Comment:
in this case, you test one value only for the cat but not for the dog, the same in other cases. You should check all age boundaries for both of them

Reply:
sorry, I don't get it
both 0 and 14 in one boundary range, less than 15. so it would be the same test if I run it for 0, 0 or 14, 14, or  3, 12 for instance

Reply:
I also don`t understand, why I should test for an incorrect data type. In this case, the function doesn`t check for input data type and doesn't raise custom exceptions. So if a user ignores type annotations and passes a string to the function, it will raise TypeError: '<' not supported between instances of 'str' and 'int'
Am I supposed to check for negative numbers?
Any "large number" is also in the same rule range. So test for a test for 25 and 125 years is basically the same test

Reply:
@AnnaBoichenko u should check an incorrect data type to test if function works correctly with incorrect data type and raises exception when it needed, the same is for other cases, so implement all required tests in readme.md file
Imagine, that u don't know how this function works inside, so u should test all possible conditions for it